src/Workspaces/Core/Portable/Workspace/Workspace_Events.cs

```diff
@@ -103,12 +103,13 @@ protected Task RaiseWorkspaceChangedEventAsync(WorkspaceChangeKind kind, Solutio
 
         static void RaiseEventForHandlers(
             EventMap.EventHandlerSet<EventHandler<WorkspaceChangeEventArgs>> handlers,
+            Workspace workspace,
```

Comment:
>workspace [](http://example.com/codeflow?start=22&length=9)

nit: might be a bit more clear if the name indicated that this is the sender of the event.

Reply:
@ToddGrun: renamed a bit, this look better?

Reply:
beautiful!